app/views/actual_db_schema/phantom_migrations/index.html.erb

```diff
@@ -51,5 +51,8 @@
         <p>No phantom migrations found.</p>
       <% end %>
     </div>
+    <% flash.each do |key, message| %>
+      <div class="flash <%= key %>"><%= message %></div>
+    <% end %>
```

Comment:
@VladislavSokov if there are too many migrations, will the user see this message? Is it ok? Please think about users